src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/Layout.php

```diff
@@ -31,9 +41,32 @@ public function execute(Varien_Event_Observer $observer)
             return $this;
         }
 
-        $layout = $observer->getEvent()->getLayout()->getUpdate();
-        $layout->addHandle('shopgate_cloudapi_default');
+        /** @var Mage_Core_Model_Layout $layout */
+        $layout = $observer->getEvent()->getLayout();
+        $layout->getUpdate()->addHandle('shopgate_cloudapi_default');
+        $this->addCustomHandle($layout);
 
         return $this;
     }
+
+    /**
+     * @param Mage_Core_Model_Layout $layout
+     */
+    private function addCustomHandle($layout)
```

Comment:
name should be plural as you are adding more than one (possibly)

Reply:
done